alvr/client_openxr/src/lobby.rs

```diff
@@ -106,11 +106,15 @@ impl Lobby {
         self.xr_session
             .sync_actions(&[(&self.interaction_ctx.read().action_set).into()])
             .ok();
+
+        // future_time doesn't have to be any particular value, just something after vsync_time
+        let future_time = vsync_time + Duration::from_millis(80);
```

Comment:
Is it intentional that this is so high?

Reply:
It's roughly about what we expect while streaming. Velocities in the lobby are not very important, it's just for displaying in debug builds

Reply:
Isn't this still the velocity that gets used for prediction tho? Or am I missing something?

Reply:
Nope. for lobby, the base time is already the one predicted to the vsync time, then we use a time after that to then calculate the velocities. And this is just for pico and vive.

Reply:
Yea but we're trying to better calculate the velocities to work around the pico bug, no? Or why are we doing this change?

Reply:
We are sharing the code between lobby and stream. The main goal of this change is for the stream but we need to handle this on the lobby too.

Reply:
I gotta learn how to read code...